packages/forms/src/UIForm/fields/Select/Select.component.js

```diff
@@ -39,32 +33,22 @@ export default function Select({ id, isValid, errorMessage, onChange, onFinish,
 				autoFocus={autoFocus}
 				className="form-control"
 				disabled={disabled}
-				onBlur={event => onFinish(event, { schema })}
-				onChange={
-					event => onChange(
-						event,
-						{ schema, value: getSelectedOptions(event.target, multiple) }
-					)
-				}
+				onChange={event => {
+					onChange(event, { schema, value: getSelectedOptions(event.target, multiple) });
+					onFinish(event, { schema, value: getSelectedOptions(event.target, multiple) });
```

Comment:
Can you init only 1 time the payload ? Store it in a local var and pass it to onChange and onFinish